src/components/gds-button/gds-button.tsx

```diff
@@ -1,18 +1,18 @@
-import { Component, Host, h } from '@stencil/core'
+import { Component, Prop, h } from '@stencil/core'
 
 @Component({
   tag: 'gds-button',
   styleUrl: 'gds-button.scss',
   shadow: true,
 })
 export class GdsButton {
+  @Prop({ reflect: true }) disabled: boolean = false
+
   render() {
     return (
-      <Host>
-        <button>
-          <slot></slot>
-        </button>
-      </Host>
+      <button class="gds-button" disabled={this.disabled}>
```

Comment:
I know using class names might not be ideal but it sounds useful if you could just import the design system and use the classes without further setup.

```scss
@use '~genero-design-system/scss/gds';

// .gds-button is now available
```

Reply:
This is nice

Reply:
I think this is the best approach anyways. Maybe you should be able to specify that the button should be an `<a>` with href